compatibility.bs

```diff
@@ -783,6 +783,35 @@ rendered as follows:
 <img style="width: 381px;height: 116px;" src="https://miketaylr.com/post/stroked-text.png" alt="image of stroked text">
 </div>
 
+<h3 id="css-property-values">CSS Property values</h3>
+
+<h4 id="touch-action">Additional <code>touch-action</code> values</h4>
+
+This section augments the <a href="https://w3c.github.io/pointerevents/#the-touch-action-css-property">
+definition of <code>touch-action</code></a> from [[!pointerevents2]] to
+<a href="https://github.com/whatwg/compat/issues/68">add</a> the <code>pinch-zoom</code> value.
+
+<pre class='propdef'>
+Name: touch-action
+Value: auto | none | [ [ pan-x | pan-left | pan-right ] || [ pan-y | pan-up | pan-down ] || pinch-zoom ] | manipulation
+Initial: auto
+Applies to: all elements except: non-replaced inline elements, table rows, row groups, table columns, and column groups.
+Inherited: no
+Computed value: Same as specified value
+Percentages: N/A
+Media: visual
+</pre>
+
+When specified, the <code>pinch-zoom</code> token enables two-finger panning and zooming of the page.
```

Comment:
Sure, done.  I don't want to try to go into a ton of detail here, but that does seem to better reflect the state of implementations today than pretending that "two" is somehow special.